docs/DEVELOPMENT.md

```diff
@@ -92,15 +92,29 @@ In order to be merged your opened pull request (PR) should pass the static analy
 
 The e2e tests jobs are executed on a variety of CcRuntime, configurations and platforms. These jobs that require confidential hardware (Intel TDX, AMD SEV, IBM SE, etc...) run on bare-metal machines and are often referred as "TEE tests". The remaining tests (a.k.a "Non-TEE") are executed on Virtual Machines (VM) deployed on-demand.
 
+This is a list of the bare-metal machines and VMs that are utilized by the project for CI testing:
+
+| Machine            | Type       | TEE Capability | Assigned Labels |
+| ---                | ---        | ---            | ---             |
+| amd-milan-kata-ci  | bare-metal | AMD SNP        | amd-milan       |
+| amd-rome-kata-ci   | bare-metal | AMD SEV        | amd-rome        |
+| s390x              | virtual    | IBM SE         |                 |
```

Comment:
I am sorry. `IBM SE` should be changed to Non-TEE.

Reply:
Done.